mne/decoding/time_gen.py

```diff
@@ -356,6 +356,7 @@ def predict(self, epochs):
         if self.test_times == 'diagonal':
             test_times = _DecodingTime()
             test_times['slices'] = [[s] for s in self.train_times_['slices']]
+            test_times['times'] = [[s] for s in self.train_times_['times']]
```

Comment:
Is `self.train_times_['times']` an array? If so I think this could just be `self.train_times_['times'].copy()[:, np.newaxis]`, it makes it clearer that you're making a copy and increasing the `ndim` of the array, assuming I'm intrepreting the list comprehension correctly.